src/Global.lua

```diff
@@ -198,6 +198,15 @@ function onObjectDrop(player_color, object)
 
 end
 
+function onPlayerAction(player, action, targets)
+    -- if the player action is "FlipOver" and 1 action card is targetted
+    if (action == 5 and #targets == 1 and targets[1].hasTag("Action")) then
```

Comment:
use `action == Player.Action.FlipOver` for self-documenting code.

Reply:
Good point!  Updated.